subprojects/analysis-java-psi/src/main/kotlin/org/jetbrains/dokka/analysis/java/parsers/DokkaPsiParser.kt

```diff
@@ -789,6 +789,16 @@ internal class DokkaPsiParser(
                     annotation.hasQualifiedName("java.lang.annotation.Documented")
                 }
             )
+                .let {
```

Comment:
JFTR: Yes, that was my initial attempt :) I tried to write a `DocumentableTransformer`, but because the annotation is sooo deep into the documentable model, the amount of nested code like `module.copy(package.copy(class.copy(extra = extra.filter.copy())))` grew beyound reasonable, so it was actually much easier and cleaner to just hardcode it like I did. 

We would indeed have 3 different places with this fix, but we knew that it would be fixed in the standard library around 1.9.20/2.0, so we wouldn't have these hacks for long, and I specifically formatted the code such that it can be easily deleted